Library/Homebrew/compat/tap.rb

```diff
@@ -0,0 +1,38 @@
+module CaskTapMigration
+  def initialize(user, repo)
+    super
+
+    return unless user == "caskroom"
+
+    # TODO: Remove this check after migration.
+    return unless repo == "tap-migration-test"
```

Comment:
Make this an if?

Reply:
How?

Reply:
`return if repo != "tap-migration-test"`

Reply:
I'm pretty sure RuboCop would complain about that.

Reply:
@reitermarkus Try it; pretty sure it doesn't 😉